setup/providers/kubernetes.md

```diff
@@ -36,7 +36,7 @@ hosted solutions:
 
 * [Google Container Engine](https://cloud.google.com/container-engine/)
 * [Azure Container
-  Service](https://docs.microsoft.com/en-us/azure/container-service/container-service-kubernetes-walkthrough)
+  Service](https://docs.microsoft.com/en-us/azure/container-service/container-service-kubernetes-walkthrough) [(codelab)](http://aka.ms/azspinlab)
```

Comment:
I don't think linking to the codelab here makes sense. This page assumes you have an existing Kubernetes cluster and you are setting up Spinnaker separately. The sentence preceding this even implies you just want to create a cluster by itself

That being said, the existing link is a pretty heavy doc page and we might be able to find a better link